packages/ag-charts-enterprise/src/series/box-plot/boxPlotSeries.ts

```diff
@@ -450,36 +452,17 @@ export class BoxPlotSeries extends CartesianSeries<
         datumSelections,
     }: _ModuleSupport.CartesianAnimationData<_ModuleSupport.SeriesNodeDataContext<BoxPlotNodeDatum>, BoxPlotGroup>) {
         const isVertical = this.direction === 'vertical';
-        datumSelections.forEach((datumSelection) => {
-            datumSelection.each((boxPlotGroup, datum) => {
-                if (isVertical) {
-                    boxPlotGroup.scalingCenterY = datum.scaledValues.medianValue;
-                } else {
-                    boxPlotGroup.scalingCenterX = datum.scaledValues.medianValue;
-                }
-            });
-            this.ctx.animationManager.animate({
-                id: `${this.id}_empty-update-ready`,
-                from: 0,
-                to: 1,
-                ease: _ModuleSupport.Motion.easeOut,
-                onUpdate(value) {
-                    datumSelection.each((boxPlotGroup) => {
-                        if (isVertical) {
-                            boxPlotGroup.scalingY = value;
-                        } else {
-                            boxPlotGroup.scalingX = value;
-                        }
-                    });
-                },
-                onStop() {
-                    datumSelection.each((boxPlotGroup) => {
-                        boxPlotGroup.scalingX = 1;
-                        boxPlotGroup.scalingY = 1;
-                    });
-                },
-            });
-        });
+
+        motion.resetMotion(datumSelections, resetBoxPlotSelectionsScalingCenterFn(isVertical));
+
+        const { from, to } = prepareBoxPlotFromTo(isVertical);
+        motion.staticFromToMotion(
+            `${this.id}_empty-update-ready`,
+            this.ctx.animationManager,
+            datumSelections,
+            from,
+            to
+        );
```

Comment:
We're missing the `onStop`

Reply:
It may change some behaviours in the situation where new animations are started before the old ones have finished. But I can't tell from looking at the code. Best to check it against the examples on the animation page.

Reply:
But that may be based on outdated understanding of the animation manager too

Reply:
👍 All looks good on the animation page from what I can see.